src/pages/amrs/[name].tsx

```diff
@@ -187,6 +168,14 @@ export async function getStaticProps({
     charging_speed: amrMatterResult.data.charging_speed,
     payload: amrMatterResult.data.payload,
   } as Props["amrfacts"];
+  const amrUsage = {
+    usage: amrMatterResult.data.usage,
+  } as {
+    usage: {
+      heading: string;
+      description: string;
+    }[];
+  };
```

Comment:
Nimm lieber die typen von oben
```suggestion
  } as Props["amrUsage"];
```